src/components/structural/header/ProjectView.js

```diff
@@ -440,7 +440,8 @@ class Project extends React.Component {
                                     <div className="row" id="user-proj" style={{ width: "100%" }}>
                                         { // Sort the users projects in alphabetical order
                                             userProjs.sort(function (a, b) {
-                                                return a.name < b.name ? -1 : a.name > b.name ? 1 : a.updateTime > b.updateTime ? -1 : a.updateTime < b.updateTime ? 1 : 0;
+                                                
+                                                return a.name.toUpperCase() < b.name.toUpperCase() ? -1 : a.name.toUpperCase() > b.name.toUpperCase() ? 1 : a.updateTime > b.updateTime ? -1 : a.updateTime < b.updateTime ? 1 : 0;
```

Comment:
Let's fix this overly long ternary to a if/else setup so that it's more readable